docker-compose.yml

```diff
@@ -22,4 +25,26 @@ services:
       - "7000:80"
     networks:
       - localdev
-    image: project-api-image:latest
\ No newline at end of file
+    depends_on:
+      - sql-server-db
+    restart: always
+    image: project-api-image:latest
+
+  sql-server-db:
+    container_name: 'sql-server-db'
+    hostname: 'TeamProject'
+    image: mcr.microsoft.com/mssql/server:2022-latest
+    ports:
+      - "1433:1433"
+    volumes:
+      - ./server/volumes/data:/var/opt/mssql/data
+      - ./server/volumes/log:/var/opt/mssql/log
+      - ./server/volumes/secrets:/var/opt/mssql/secrets
+    env_file:
+      - server/.env
+    restart: always
```

Comment:
Please add sql-server-db to the same network as other containers (localdev) to be consistent